### ISSUE TITLE
Transaction timestamp is higher than block timestamp

### ISSUE THREAD

**Issue by honeycrypto** [semuxproject/semux-core#162]
Here is an example: https://semux.info/explorer/block/1371015 

This block includes 91 Tx and some of them (from `savel` validator) have timestamp 22 May 2019 12:00:19 GMT. 
The block timestamp is 22 May 2019 12:00:05 GMT. So these Txs are from the future (14 seconds ahead). 

Shouldn't validator reject these TXs in that specific block with a lower timestamp?

**Comment by orogvany** [comment]
I think there is some allowed timeshift.  We could ignore/not propose any in future, but apart from being visually distracting, any reason to deny/delay tx from client who has wrong local time set?

**Comment by honeycrypto** [comment]
Probably there is only visual issue for transactions from the future. But I now realised that I'm using `tx.timestamp` for Last Activity in explorer's rich list. So someone can spam the network with transactions with timestamp months/years ago. Can this work? At what point validators will reject transaction as too old (even if valid)? 
I mean if the 'very old' tx has been added to mempool just now.
